.travis.yml

```diff
@@ -4,21 +4,13 @@ jobs:
   include:
     - node_js: "12"
       env: CXX=g++-4.8 TCHANNEL_TEST_CONFIG= NPM_TEST_SCRIPT=travis
-    - node_js: "5"
```

Comment:
10 is still LTS we can test on 10 and 12. I think 8 left maintenance so it’s not needed